site/_docs/windows.md

```diff
@@ -10,12 +10,14 @@ knowledge and lessons that have been unearthed by Windows users.
 
 ## Installation
 
-Julian Thilo has written up instructions to get
-[Jekyll running on Windows][windows-installation] and it seems to work for most
-people. The instructions were written for Ruby 2.0.0, but should work for later
-versions [prior to 2.2][hitimes-issue].
+A quick way to install Jekyll it to follow the [installation instructions by David Burela](https://davidburela.wordpress.com/2015/11/28/easily-install-jekyll-on-windows-with-3-command-prompt-entries-and-chocolatey/):
```

Comment:
Should it be "to install Jekyll **is** to follow" instead of "to install Jekyll **it** to follow" ?